src/calendar/calendar.jsx

```diff
@@ -66,9 +65,12 @@ export default class Calendar extends React.Component {
   }
 
   static defaultProps = {
-    weekStartsOn: 1,
-    mode: SINGLE_MODE,
     blockClassName: BLOCK_CLASS_NAME,
+    daysOfWeek: DAYS_OF_WEEK,
```

Comment:
*Important*: `Calendar` component is responsible for set default props (single source of truth)